rego/rules/arm/app_service/min_tls_version.rego

```diff
@@ -37,20 +37,34 @@ input_type = "arm"
 
 resource_type = "MULTIPLE"
 
-resources = fugue.resources("Microsoft.Web/sites")
+sites := fugue.resources("Microsoft.Web/sites")
+configs := fugue.resources("Microsoft.Web/sites/config")
 
-is_invalid(resource) {
-	resource.TODO == "TODO" # FIXME
+min_tls_version := "1.2"
+
+valid_via_config := {id |
+	c := configs[_]
+	c.name == "web"
+	c.properties.minTlsVersion >= min_tls_version
```

Comment:
Nitpick but we could consider adding this to a library so we can compare versions as arrays rather than strings:
```
parse_version(str) = ret {
  ret := [to_number(p) | p = regex.find_n(`[0-9]+`, str, -1)[_]]
}
```

Reply:
Thanks! Yeah that is a nice improvement. String comparison works with the way that TLS versions have historically been specified, but your suggestion is more future proof.

I was hoping to use OPA's `semver.compare()` here, but it only works with full semvers.

Reply:
Fixed in 3fc94f8